pkg/app/rds_postgres.go

```diff
@@ -141,23 +149,41 @@ func (pdb *RDSPostgresDB) Install(ctx context.Context, ns string) error {
 	if err := kube.WaitOnDeploymentReady(ctx, pdb.cli, pdb.namespace, pdb.bastionDebugWorkloadName); err != nil {
 		return errors.Wrapf(err, "Failed while waiting for deployment %s to be ready, app: %s", pdb.bastionDebugWorkloadName, pdb.name)
 	}
+
+	pdb.vpcID, err = getVpcIdForRDSInstance(ctx, ec2Cli)
+	if err != nil {
+		return err
+	}
+
+	// describe subnets in the VPC
+	resp, err := ec2Cli.DescribeSubnets(ctx, pdb.vpcID)
+	if err != nil {
+		return errors.Wrapf(err, "Failed to describe subnets")
+	}
+
+	// Extract subnet IDs from the response
+	var subnetIDs []string
+	for _, subnet := range resp.Subnets {
+		subnetIDs = append(subnetIDs, *subnet.SubnetId)
+	}
+	// create a subnetgroup with subnets in the VPC
+	subnetGroup, err := rdsCli.CreateDBSubnetGroup(ctx, fmt.Sprintf("%s-subnetgroup", pdb.name), "kanister-test-subnet-group", subnetIDs)
+	if err != nil {
+		return errors.Wrapf(err, "Failed to create subnet group")
+	}
+	pdb.dbSubnetGroup = *subnetGroup.DBSubnetGroup.DBSubnetGroupName
```

Comment:
I see this as a common code in aurora and rds files. Can we move it to a helper method?